src/components/dialogs/incorporate-now-error.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach">
+    <v-card>
+      <v-card-title id="dialog-title">Unable to Incorporate Now</v-card-title>
+
+      <v-card-text id="dialog-text">
+        <!-- display message -->
+        <div class="general-error">
+          <p>Unable to incorporate now. Please cancel or try again.</p>
```

Comment:
1. Didn't the UI team want to change everything to "Register now"?
2. Is this modal also used for registrations? Maybe the text should be generalized.

-> New ticket if needed, please.

Reply:
I was just looking at this Sev. I noticed that the `showRegisterButton` is always false currently. Hence, for SPs/GPs, we're showing the Incorporate your Business button. 

I'm not sure about the first point. I'll double check with Yui. As for the second point, yes you're right. 

Let me double check with the UX team about that.